packages/core/src/tracing/index.js

```diff
@@ -203,13 +239,20 @@ exports.enableSpanBatching = function enableSpanBatching() {
 exports._getAndResetTracingMetrics = function _getAndResetTracingMetrics() {
   return {
     pid:
+      // TODO: This is not instrumented yet -> /nodejs-sensor/packages/collector/src/pidStore/index.js
```

Comment:
Not instrumented yet? Or did you mean not _typed_ yet?

Reply:
Oops, I was probably thinking about something else.